draft-ietf-ppm-dap.md

```diff
@@ -1404,9 +1404,9 @@ Clients retrieve the HPKE configuration from each Aggregator by sending a GET to
 `{aggregator}/hpke_config`.
 
 An Aggregator responds with an `HpkeConfigList`, with media type
-"application/dap-hpke-config-list". The `HpkeConfigList` contains one or more
-`HpkeConfig`s in decreasing order of preference. This allows an Aggregator to
-support multiple HPKE configurations and multiple sets of algorithms
+"application/dap;message=hpke-config-list". The `HpkeConfigList` contains one or
```

Comment:
Should there be a space here? Here and below.
```suggestion
"application/dap; message=hpke-config-list". The `HpkeConfigList` contains one or
```

Reply:
Either is valid according to RFC 9110
https://www.rfc-editor.org/rfc/rfc9110.html#section-8.3.1